scripts/bank/preview_outcome.py

```diff
@@ -43,8 +43,8 @@
 </bank>
 """.strip())
 (sandbox_path/"outcome").mkdir()
-(sandbox_path/"library").mkdir()
-shutil.copy(Path("exercises")/"library"/"common.sage", sandbox_path/"library"/"common.sage")
+# (sandbox_path/"library").mkdir()
+# shutil.copy(Path("exercises")/"library"/"common.sage", sandbox_path/"library"/"common.sage")
```

Comment:
For now I've just removed the lines as we don't have a `common.sage` for calculus. We can add these back if we do create one later.